packages/node_modules/@webex/plugin-meetings/src/meeting/request.js

```diff
@@ -98,23 +99,27 @@ export default class MeetingRequest extends StatelessWebexPlugin {
       body.moderator = moderator;
     }
 
+    if (permissionToken) {
+      body.permissionToken = permissionToken;
+    }
+
     if (pin !== undefined) {
       body.pin = pin;
     }
 
     if (locusUrl) {
       url = `${locusUrl}/${PARTICIPANT}`;
     }
-    else if (sipUri || meetingNumber) {
+    else if (inviteeAddress || meetingNumber) {
       try {
         await this.webex.internal.services.waitForCatalog('postauth');
         url = `${this.webex.internal.services.get('locus')}/${LOCI}/${CALL}`;
         body.invitee = {
-          address: sipUri || `wbxmn:${meetingNumber}`
+          address: inviteeAddress || `wbxmn:${meetingNumber}`
         };
       }
       catch (e) {
-        LoggerProxy.logger.error(`Meeting:request#joinMeeting ${sipUri ? 'sipUri' : 'meetingNumber'} --> ${e}`);
+        LoggerProxy.logger.error(`Meeting:request#joinMeeting ${inviteeAddress ? 'inviteeAddress' : 'meetingNumber'} --> ${e}`);
```

Comment:
in case when `inviteeAddress` is undefined it would be good to log the `meetingNumber` value

Reply:
done